hubmap/dataportal/management/commands/_metadata_import.py

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env python3
```

Comment:
That sounds reasonable, but the leading underscore tells Django "despite being in this directory, this Python file does not expose a `manage.py` command". It would be reasonable to have a single file instead of two, but I kind of liked the idea of separating "big ball of import code" and "Django entry point".